webpack/shared.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
 				'./client/checkout/woopay/express-button/index.js',
 			'woopay-direct-checkout':
 				'./client/checkout/woopay/direct-checkout/index.js',
+			cart: './client/cart/index.js',
```

Comment:
The tracking implementation worked on the Blocks cart page by injecting it in the Blocks checkout entry point, but I had to enqueue a new script because apparently we don't have a script that's enqueued in the short code cart. With this approach, I could centralize the tracking implementation for both cart pages in a single script.